device-types/Cisco/C9800-CL.yaml

```diff
@@ -0,0 +1,18 @@
+---
+manufacturer: Cisco
+model: C9800-CL
+slug: cisco-c9800-cl
+part_number: C9800-CL
+u_height: 1
+is_full_depth: false
+is_powered: false
```

Comment:
Why do you have this as not powered?

Reply:
Because they are VMs, so it is really the ESXI host (eg. HP server) which has power..
My issue is that this VM carries traffic from APs, so it really is a networking device, with primary IPs, interfaces etc.

Reply:
C9800 removed in latest commit.